src/middleware/authentication_middleware.ts

```diff
@@ -11,5 +11,7 @@ export const authenticationMiddleware = (req: Request, res: Response, next: Next
         returnUrl: BASE_URL + CHECK_SAVED_APPLICATION
     };
 
-    return acspProfileCreateAuthMiddleware(authMiddlewareConfig)(req, res, next);
+    return isActiveFeature(FEATURE_FLAG_VERIFY_SOLE_TRADER_ONLY) === true
```

Comment:
If you are calling `isActiveFeature()` then you don't need check `=== true`

Reply:
done, thanks